drivers/flash/soc_flash_nrf_rram.c

```diff
@@ -58,6 +58,30 @@ BUILD_ASSERT((PAGE_SIZE % (WRITE_LINE_SIZE) == 0),
 	     "erase-block-size must be a multiple of write-block-size");
 #endif
 
+#ifdef CONFIG_TRUSTED_EXECUTION_NONSECURE
+/* non-secure images cannot call nrf_rramc_config_set because
+ * NRF_RRAMC_NS does not exist.
+ *
+ * Instead, when TF-M boots, it will configure RRAMC with this static
+ * configuration:
+ *
+ * nrf_rramc_config_t config = {
+ *   .mode_write = true,
+ *   .write_buff_size = WRITE_BUFFER_SIZE
+ * };
+ *
+ * nrf_rramc_ready_next_timeout_t params = {
+ *   .value = CONFIG_NRF_RRAM_READYNEXT_TIMEOUT_VALUE,
+ *   .enable = true,
+ * };
+ *
+ * For more details see NCSDK-26982.
+ */
```

Comment:
No conditional comments, looks strange, like leftover. Just do comment above the `#if !defined()`, should be enough

Reply:
OK